packages/server/src/DataSource.ts

```diff
@@ -89,3 +79,15 @@ export function getDataSource(): DataSource {
     }
     return appDataSource
 }
+
+const getDatabaseSSLFromEnv = () => {
+    if (process.env.DATABASE_SSL_KEY_BASE64) {
+        return {
+            rejectUnauthorized: false,
+            ca: Buffer.from(process.env.DATABASE_SSL_KEY_BASE64, 'base64')
```

Comment:
@Jaredude have you tested it? 

I think it should be like this:

`Buffer.from(process.env.DATABASE_SSL_KEY_BASE64, 'base64').toString('utf-8')`

Reply:
@Jaredude WDYT?

Reply:
I don't see anything in the pg docs or mysql2 docs requiring the data to be utf8

Reply:
@Jaredude ok thanks , highly appreciated

One last question, can you please explain how does it work when we provide only ssl:true but no certificate?

Reply:
@niztal Great question... Hope this helps: the reason that ssl: true works without supplying a ca or cert is that each library will reference a trust store(s) to verify the certificate coming from the server. The best analogy I can think of would be visiting a website over https instead of http. The web browser validates the ssl connection by checking to make sure the authority of the certificate is indeed valid. If it's not, you get the screen showing you the certificate is not valid. The pg and mysql2 libraries have their own trust stores that they use, and that allows a simple ssl: true property to work. In the case of the mysql2 lib, certs from major providers are directly in the managed source code for mysql2. You can view them under the files in node_modules/mysql2.

Reply:
Wow amazing explanation @Jaredude thank you so much